Cargo.lock

```diff
@@ -3695,7 +3729,7 @@ checksum = "3db6840b7adcfd2e866c79157cc890ecdbbc1f739607134039ae64eaa6c07e24"
 dependencies = [
  "clap 2.34.0",
  "owo-colors",
- "parking_lot",
+ "parking_lot 0.11.2",
```

Comment:
When you see the lock file updated with a version like this, it's a hint that there are now more than one version of this crate required in the dependency graph.

It looks like there might be another dependency in the graph that still depends on the old version of `parking_lot` and `dashmap`. We should check whether or not the other crates that require these deps are within the fuel ecosystem so that we can keep the crate dependency count low and avoid pulling in more crates than necessary.

Reply:
Ok cheers yeah that makes sense, Seems like the `uwuify` crate is using the older version of parking_lot, happy to open up a PR to `uwuify` to update the dep. 

Might just leave `dashmap` at 4.0.2 since its being used in the new version of `lspower` and `http-client`